iphone/Classes/UIModule.m

```diff
@@ -599,30 +598,45 @@ -(NSString *)ATTRIBUTE_LETTERPRESS_STYLE
 {
     return NSTextEffectLetterpressStyle;
 }
--(NSNumber*)ATTRIBUTE_LINE_BREAK_BY_WORD_WRAPPING
+-(NSNumber*)ATTRIBUTE_PARAGRAPH_STYLE_LINE_BREAK_BY_WORD_WRAPPING
```

Comment:
This would be a breaking change. We should deprecate them like in the above example + remove it in the next major release.